### ISSUE TITLE
@paulnara are you still experiencing this issue? If yes, could you share some more details as we have not been able to reproduce this.

### ISSUE THREAD

**Issue by csanjay99** [Azure/azure-functions-core-tools#3109]
Hi, I see this has been closed but I would like to open it again. This is a serious issue and it needs to be fixed. I am getting the same error as @Kaushik27. Tried all workarounds  mentioned by @Kaushik27 still nothing seems to work. The error is same "self signed certificate in chain". Please so something ab out it.

**Comment by Kaushik27** [comment]
I do agree with you. But Microsoft is very negligent around this. They will take humongous amount of time to reply @csanjay99.
And by that time you would have already found an alternative or changed your approach itself.

**Comment by michaelpeng36** [comment]
@csanjay99 Are you using a proxy server? Could you also clarify if this issue is present for both Core Tools V3 and Core Tools V4?

**Comment by csanjay99** [comment]
Yes, Netskope was installed on my computer. I found this a couple of days after posting here. Sorry to not update it here. I went on its website and did what I was asked to in order for the certificates to work. @Kaushik27 You should also find out if you have any proxy server that is running under the belly and might not be so obvious. Check for any program like Netskope, Zscaler or the likes of cisco installed in your computer. They are the culprit. Go on their official site and using the documentation you can get rid of this self signed certificate error. This was helpful for me as I had netskope installed. 

https://docs.netskope.com/en/configuring-cli-based-tools-and-development-frameworks-to-work-with-netskope-ssl-interception.html 

Thank you all, Microsoft can close this from my end.
